### ISSUE TITLE
Ivy

### ISSUE THREAD

**Issue by merobal** [formio/angular-formio#505]
@travist I debugged the issue with Ivy today.
The `auth`, `manager`, `grid`, and `resource` modules are packed individually but contain references for each other, eg. `manager` imports `auth`. Ivy detects the modules as standalone packages and tries to manage async, but it reaches `auth` through `manager` as well, then throws an error since there is a lock file indicates it already processing it.
The solution would be re-structuring the lib to having only 1 entry point for each module for Ivy.

**Comment by travist** [comment]
Thanks for this @merobal!  We will add this to our developer backlog.

**Comment by merobal** [comment]
Do you have any update/plans related to Ivy?

**Comment by wag110894** [comment]
This item has been resourced. We will let you know once this has been resolved.

**Comment by merobal** [comment]
do you have any update?

**Comment by wag110894** [comment]
@merobal, thanks for reaching out. This item is on one of our developers list. We will let you know once this issue has been resolved.

**Comment by chaixdev** [comment]
couple of months later, any update on Ivy support?

**Comment by wag110894** [comment]
Hello @chaixdev, This item is still on our developer's list. We will let you know once this issue has been resolved.

**Comment by arjel** [comment]
Hello, sorry to bother you guys ... any update on Ivy support yet?
Thanks!

**Comment by travist** [comment]
This maybe fixed with our latest upgrade @ ```@formio/angular``` repo. Have you tried that version @arjel?

**Comment by arjel** [comment]
@travist I tried on v4.11.0 and still had problems. I can try to update/reinstall and see if it's ok. Thank you!

**Comment by RyanHow** [comment]
Hey guys. Just found this issue. Just reporting that it is a major showstopper for us and a decision-maker or breaker when considering this library.

(Just mentioning because if I was maintaining then I'd like to know why people are using/non using my library)

**Comment by SusanneOlsenTrifork** [comment]
Hey, 
Just tried to upgrade to Angular 12, and we're now forced to use Ivy.... How far is the implementation of Ivy support @travist ?

When building our project we receive errors like:
```
Error: Tried to write C:/**/node_modules/angular-formio/formio.config.d.ts.__ivy_ngcc_bak with an ngcc back up file but it already exists so not writing, nor backing up, C:/**/node_modules/angular-formio/formio.config.d.ts.
This error may be caused by one of the following:
* two or more entry-points overlap and ngcc has been asked to process some files more than once.
  In this case, you should check other entry-points in this package
  and set up a config to ignore any that you are not using.
* a previous run of ngcc was killed in the middle of processing, in a way that cannot be recovered.
  In this case, you should try cleaning the node_modules directory and any dist directories that contain local libraries. Then try again.
```

Though the application builds and is able to run. 
Besides these errors we are experiencing problems with emitting updated data through the FormioBaseComponent.refresh event.

**Comment by JonasDev99** [comment]
@travist Has support for angular been dropped?

**Comment by brendanbond** [comment]
@JonasDev99 No. Are you receiving the above errors or just asking generally?

**Comment by daneformio** [comment]
Ticket for internal tracking: FIO-3190
